app/src/main/res/drawable/custom_switch.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We also voiced some issues with the custom_switch. Personally, I would prefer if it kept with the material design/android style as much as possible